charts/prometheus-elasticsearch-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 4.4.0
+version: 4.5.0
```

Comment:
update to 4.6.0 please

Reply:
bumped the version to `4.6.0` as requested